js_machine_front/src/i18n/messages.ts

```diff
@@ -0,0 +1,57 @@
+export const messages: {[index: string]: any} = {
```

Comment:
no any, you can cretae interface {['en-Us']: {[index: string]: string}}

Reply:
fixed